### DISCUSSION TITLE
Tags in Status field

### DISCUSSION THREAD

**Discussion by 4C3T** [airdcpp/airdcpp-windows#100] [Q&A] [answered]
Hi. Not sure where to ask, but does anyone know what these tags stand for?:
"[M][S][T][Z] Downloaded 1,25 GiB (32.7%) in 4:26:14"

Others appears as
"[Z] Uploaded 1,22 GiB (84.1%) in 5:08:36"
"[S][Z] Uploaded 1,22 GiB (77.2%) in 4:27:30" 
"[M][S][Z] Uploaded 1,89 GiB (95.5%) in 1:19:10"

**Comment by Sopor** [comment, marked as answer]
[T] = TTH tree available (file is being verified during the download)
[Z] = transfer compressed using ZLIB
[R] = rollback used for continuation
[S] = trusted encrypted transfer
[U] = untrusted encrypted transfer
[P] = partial file sharing source
[C] = chunked transfer
[M] = other client supports multiple connections per users for transfers

**Comment by 4C3T** [comment]
Nice, thanks

**Reply by Sopor** [reply]
`[C]` will also indicate resume if a user has been disconnected or if he has changed source.

If you see `[C]` on every file the user is downloading, the user has enable segmented downloads, if it is on a single file it is probably resume.
